### DISCUSSION TITLE
[Idea] Add password protection to GLPI-Agent for Android

### DISCUSSION THREAD

**Discussion by xcassanello** [glpi-project/glpi-agent#481] [Ideas]
### Your idea

Hello,

I think it would be awesome if GLPI Agent could be password protected against griefing, Android version above all. 

You can easily protect software in Windows through user permissions, GPO, Intune, etc... But on android things are different and in IT you have 2 choices: 
- Give user some freedom with their devices, which is good since it engages the user with the business.
- Block absolutely any additional interaction that is not business related, this dissengage users to stop using their devices even after work.

The issue with choice 1 is that user can indeed delete the app, or even change configuration if the MDM blocks the uninstall. So a password protection for configuration in Android can be a sweet bonus for IT Administrators.

To let my imagination fly, this can even be configured in the server and devices when connect with GLPI Agent push this kind of policies from the server.

**Comment by g-bougard** [comment]
Hi @xcassanello 

the android agent has [its own project](https://github.com/glpi-project/android-inventory-agent), but I think @stonebuzz still can answer you.
